src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        if (userService.findByEmail(email) != null
+                && userService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
```suggestion
        } 
        return false;
```
we don't need else after return